### DISCUSSION TITLE
The bug of Lotus transferring coins to the market

### DISCUSSION THREAD

**Discussion by duge996** [filecoin-project/lotus#12789] [General Q&A] [answered]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [X] lotus daemon - chain sync
- [ ] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [X] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
lotus version 1.30.1-dev+mainnet+git.ba0bf74e5.dirty.amd.Ubuntu.20.04
```

### Repro Steps

```
lotus wallet market  add --h
NAME:
   lotus wallet market add - Add funds to the Storage Market Actor

USAGE:
   lotus wallet market add [command options] <amount>

OPTIONS:
   --from value, -f value     Specify address to move funds from, otherwise it will use the default wallet address
   --address value, -a value  Market address to move funds to (account or miner actor address, defaults to --from address)
   --help, -h                 show help
```

### Describe the Bug

I made an error executing the command. The correct command is 
```lotus wallet market  add --from f3rvhgu5izn2qz6pa2so474ojywxwxzryt4rnxpkap74h4xyvaywithwnfmaorqwvz2vm2f4v2spcbw5x7om3q --address f03238633 100```

But I encountered an unexpected error while executing,The parameters are incorrect, but the command was executed successfully. fil has been transferred to the f05 address. Can this be transferred back to my wallet
```lotus wallet market  add --from f3rvhgu5izn2qz6pa2so474ojywxwxzryt4rnxpkap74h4xyvaywithwnfmaorqwvz2vm2f4v2spcbw5x7om3q 100 --address f03238633 100```

### Logging Information

```text
message id
://filfox.info/en/message/bafy2bzaceadbznt5cwll3twqfpaddx3dligirpkmhe5yfg7de4t2ufnjlttq2?t=1
```

**Comment by rjan90** [comment, marked as answer]
Hey @duge996. Can you check `lotus wallet list --market`, that should output the market balance of your different addresses.

Then you can use `lotus wallet market withdraw` to withdraw your market balance from a specific address:

```
lotus wallet market withdraw -h
NAME:
   lotus wallet market withdraw - Withdraw funds from the Storage Market Actor

USAGE:
   lotus wallet market withdraw [command options] [amount (FIL) optional, otherwise will withdraw max available]

OPTIONS:
   --wallet value, -w value   Specify address to withdraw funds to, otherwise it will use the default wallet address
   --address value, -a value  Market address to withdraw from (account or miner actor address, defaults to --wallet address)
   --confidence value         number of block confirmations to wait for (default: 5)
```

**Comment by rjan90** [comment]
I will convert this issue into a discussion, as it is more of a question than an actual issue.

**Reply by rvagg** [reply]
There is a usage issue here but I've captured it in https://github.com/filecoin-project/lotus/issues/12790
